packages/frontend/src/components/UserSettings/InvitesModal/index.tsx

```diff
@@ -83,7 +83,12 @@ const InvitesModal: FC<{
                             w="43%"
                             {...form.getInputProps('email')}
                         />
-                        <Group spacing="xs">
+                        <Group
+                            spacing="xs"
+                            style={{
```

Comment:
```suggestion
                            pb={form.errors.email ? 'sm' : 0}
```
how does it look @jogeshwar01 ?

Reply:
@joaoviana Thank you! This is definitely a cleaner syntax. But I don't think we can use the default spacing options of mantine for the value.
These have pixel values of 12px(0.75rem), 16px (1rem), 20px(1.25rem) but we need them to be 18px.

![image](https://github.com/lightdash/lightdash/assets/85165953/36d795d4-0bd7-46e3-a8a6-944f9d103eae)

Reply:
We can use calc and var from Mantine if we want to use these options. Basically multiplying the sm by 1.5 to get 18px.
Like this - `pb={form.errors.email ? "calc(var(--mantine-spacing-sm)*1.5)" : 0}`

Reply:
Thanks for the report @jogeshwar01! I've also had a closer look on this, and we actually don't need any specific numbers to address this issue. 

If there's an error, center the containers. If there isn't, align them to the end.

Here is the diff that you can implement on this PR, which will then avoid the `padding` approach and solely rely on flex alignment 👍 

<img width="1256" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/b25937b7-ae8d-4f1e-8b91-00f6803fcabd">